site/src/docs/components/table/index.mdx

```diff
@@ -72,16 +73,18 @@ export const TableExample = () => {
   - On smaller screens, it may be beneficial to consider other presentation formats for table data. For example, a table can be broken into cards.
 
 #### Interactable tables (React)
+
 - HDS Tables support sorting. When a column is set to be sortable, a sorting button appears after its header.
   - The sorting icon type can be changed depending on the type of data in the column.
-- Only use sorting when it is necessary or otherwise beneficial to the user. 
+- Only use sorting when it is necessary or otherwise beneficial to the user.
 - HDS Table's rows can be made selectable. In this case, HDS automatically provides actions for selecting and deselecting all rows.
 - If your table includes actions that can affect multiple rows (e.g. deleting selected rows or exporting a CSV file), these actions must be placed above the table. For more information, see [Selecting rows](#selecting-rows) and [Custom actions](#custom-table-actions) variant examples below.
 
 ### Variations
 
 #### Default
-The default HDS table variant includes horizontal lines. The header row can be either dark or light depending on the brand colour used as a background. Whether you need a black or white text colour can be checked from [HDS Colour documentation page](/design-tokens/colour#brand-colour-combinations).
+
+The default HDS table variant includes horizontal lines. The header row can be either dark or light depending on the brand colour used as a background. Whether you need a black or white text colour can be checked from [HDS Colour documentation page](/foundation/design-tokens/colour#brand-colour-combinations).
```

Comment:
We do not have #brand-colour-combinations. Maybe it is #brand-colors ? 🤷

Reply:
Found it! It was under accessibility in design tokens.